lib/configuration/configuration_parser.js

```diff
@@ -0,0 +1,50 @@
+const Logger = require('../logger');
+const fsPromises = require('fs/promises');
+const clientConfiguration = require('./client_configuration');
+
+exports.parse = async function (configFilePath) {
+  const configFile = await tryToFind(configFilePath);
```

Comment:
can we name this function just `findConfigFile` or `getConfigFile`?

Reply:
changed to getConfigFile